tests/textcase/lsp/lsp_spec.lua

```diff
@@ -1 +1,35 @@
-describe("LSP", function() end)
+local test_helpers = require("tests.test_helpers")
+local textcase = require("textcase")
+
+local cur_dir = vim.fn.expand("%:p:h")
+
+---@param path string
+function M.read_file(path)
+  local fd = vim.loop.fs_open(path, "r", 438)
+  local fstat = vim.loop.fs_fstat(fd)
+  local contents = vim.loop.fs_read(fd, fstat.size, 0)
+  vim.loop.fs_close(fd)
+  return contents
+end
+
+describe("LSP", function()
+  describe("Rename", function()
+    before_each(function()
+      textcase.setup({})
+    end)
+
+    it("Should work", function()
+      local path = cur_dir .. "/tests/textcase/lsp/fixtures/component-camel-case.tsx"
+      vim.print(path)
+      local cmd = " silent exe 'e " .. path .. "'"
+      vim.cmd(cmd)
+      test_helpers.execute_keys("/variableToBeTested<CR>")
+
+      vim.cmd([[
+        lua require('textcase').lsp_rename('to_snake_case')
```

Comment:
I still don't know why it fails to modify the buffer 🤔 

<img width="991" alt="image" src="https://github.com/johmsalas/text-case.nvim/assets/6575405/0495a86a-db24-4643-9f66-484baa41c2d4">

Reply:
I tried non-healess Plenary and it works 🤔 🤔

Reply:
@peterfication do you have any idea?

Reply:
Jumping to the word is successful btw, if I do /variableToBeTested<CR>diw, it deletes the first instance of the variable

Reply:
Actually, I have some more ideas to debug this, I can debug directly the method. My guess is it is failing silently in the LSP invoke, and there is a way of figuring out https://github.com/johmsalas/text-case.nvim/pull/60

Reply:
I tried some stuff but I couldn't find anything that worked with the LSP so far.

Reply:
I found the issue. Hadn't found the solution but it is because a busy work day. Will be able to submit later